packages/neon_framework/packages/account_repository/lib/src/account_repository.dart

```diff
@@ -299,8 +315,12 @@ class AccountRepository {
 
     _accounts.add((active: active, accounts: accounts));
 
+    for (final callback in _beforeLogOutCallbacks) {
+      await callback(account!);
+    }
```

Comment:
I think that running these callbacks in parallel should be ok.
The account_repo can't guarantee execution order anyway.

Please use a `Future.wait`